vnext/ReactUWP/Modules/Animated/NativeAnimatedNodeManager.cpp

```diff
@@ -351,5 +455,13 @@ TransformAnimatedNode *NativeAnimatedNodeManager::GetTransformAnimatedNode(
   }
   return static_cast<TransformAnimatedNode *>(nullptr);
 }
+
+TrackingAnimatedNode *NativeAnimatedNodeManager::GetTrackingAnimatedNode(
+    int64_t tag) {
+  if (m_trackingNodes.count(tag)) {
+    return m_trackingNodes.at(tag).get();
+  }
+  return static_cast<TrackingAnimatedNode *>(nullptr);
```

Comment:
Was this cast not implicit? Why do you need it?

Reply:
I think i copied this from a lambda definition where it is needed, but in this definition it is not. thanks!

---
In reply to: [324895296](https://github.com/microsoft/react-native-windows/pull/3041#discussion_r324895296) [](ancestors = 324895296)